### ISSUE TITLE
Tap_action: toggle rather tap_action: more-info?

### ISSUE THREAD

**Issue by mrand** [benct/lovelace-multiple-entity-row#32]
This seems slightly different than https://github.com/benct/lovelace-multiple-entity-row/issues/31, but if you don't consider it so, my apologies - please close.  

I have a z-wave lock that I would like to show the changed_by_name attribute, and your card works well for that.  But in doing so, the default UNLOCK button (which locks/unlocks directly) is replaced with a state (LOCKED) which will pop-up the more-info with the UNLOCK button.

The top row is the "normal" entity behavior... you can simply touch the word UNLOCK and it'll unlock.  It would be my preference to keep it that way - is there a way to with multiple-entity-row that I'm overlooking?

The closest I could figure out is adding toggle: true, which works but I'd rather not use the ambiguous toggle switch for this.

![image](https://user-images.githubusercontent.com/435987/73555498-59ce9680-4413-11ea-990d-d7a9529de621.png)

```
          - entity: lock.front_door_lock_9
            secondary_info: last-changed
            toggle: true
            type: custom:multiple-entity-row
            primary:
              attribute: changed_by_name
              name: Last unlocked
```

**Comment by benct** [comment]
The top row you have there is handled in a special way by HA, which is not fully supported in this card. I could add support for customizable tap actions, which can toggle the locked state for you, but it will still display the `Locked` or `Unlocked` state value, not `Unlock` or `Lock` as the top row does.

I'll probably implement that together with the issue you referred to, but yes, they are slightly different :)

**Comment by mrand** [comment]
> but it will still display the `Locked` or `Unlocked` state value, not `Unlock` or `Lock` as the top row does.

That should work.  Or maybe I could use a template to change the wording.  Thanks!

**Comment by benct** [comment]
Standard `tap_action` configuration with `toggle` action implemented in v3.0.0. See readme for more info.

**Comment by mrand** [comment]
Thanks for the update!  Unfortunately I can't get it to work after upgrading to 3.0:

```
          - entity: lock.front_door_lock_9
            type: custom:multiple-entity-row
            toggle: false
            tap_action:
              action: toggle
```

It shows the lock state, as you said it would - but when I click it, nothing happens.
If I change it to action: more-info, then I get more-info pop-up.
If I set toggle: true, then I get the slider and it operates the lock as expected.

I can open another issue and/or help with debugging if you want.  I'm still on 0.104 for the moment, if that matters.

**Comment by benct** [comment]
Hmm, there might be an issue with the HA toggle service call that doesn't support all domains.. does it work if you try the following config?
```
- entity: lock.front_door_lock_9
  type: custom:multiple-entity-row
  toggle: false
  tap_action:
    action: call-service
    service: lock.toggle
    service_data:
      entity_id: lock.front_door_lock_9
```

**Comment by mrand** [comment]
It's actually more simple than that: there is no lock.toggle service!   The
only services are lock.lock and lock.unlock.

The standard entity (which shows the state, as I mentioned above) does
toggle it. Do they make a special case for locks and call the
appropriate service depending on state, or do they have some other way of
doing it?

**Comment by mrand** [comment]
I don't know if this is helpful at all: https://github.com/custom-cards/button-card/commit/edb419718abd631b8a3e9b03da5f1b4d0560dac6

**Comment by benct** [comment]
The toggle service for both `lock` and `cover` should now be fixed in version `3.5.0`. Sorry for taking so long!
